corehq/apps/users/views/__init__.py

```diff
@@ -1164,6 +1172,9 @@ def post(self, request, *args, **kwargs):
                                          primary_location_id=data.get("primary_location", None),
                                          program_id=data.get("program", None),
                                          assigned_location_ids=data.get("assigned_locations", None),
+                                         profile=profile,
+                                         tableau_role=data.get("tableau_role", None),
+                                         tableau_group_ids=data.get("tableau_group_ids", None)
```

Comment:
nice. I was going to ask you about this in https://github.com/dimagi/commcare-hq/pull/34618 but you've got it covered.